src/resolv/async_resolv.h

```diff
@@ -28,6 +28,9 @@
 
 #include <netdb.h>
 #include <ares.h>
+#ifndef MIN
+#include <sys/param.h>
+#endif
```

Comment:
On July 14, 2016 at 1:40:29 pm +02:00, lslebodn <<notifications@github.com>> wrote:

> In src/resolv/async_resolv.h <https://github.com/SSSD/sssd/pull/4#discussion_r70790787>:
> > @@ -28,6 +28,9 @@
> >
> > #include <netdb.h>
> > #include <ares.h>
> > +#ifndef MIN
> > +#include <sys/param.h>
> > +#endif
> 
> Is there a reason why it need to be included conditionally?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/SSSD/sssd/pull/4/files/bd4c5186e85619c0d6fdc36690b311734e552fe4#r70790787>, or mute the thread <https://github.com/notifications/unsubscribe/AAipUnHsM0tBypy9IYoXa9Acpz_dt2Bpks5qViAtgaJpZM4JMLmi>.
> 
> 
> 
> 
> Not really, the reason why I include it conditionally is to avoid a potential macro redefinition conflict if MIN is already defined, which it isn't on musl but could be in for example glibc.

Mvh
Daniel Kuehn, 073-181 577 2
Data@UrService
Hemsida: <http://www.dataaturservice.se>